scripts/sysman/memory.yml

```
@@ -221,7 +221,7 @@ params:
       desc: "[in] Handle for the component."
     - type: $s_mem_bandwidth_t*
       name: pBandwidth
-      desc: "[in,out] Will contain the current health, free memory, total memory size."
+      desc: "[in,out] Will contain total number of bytes read and written from and to memory, resp., as well as current maximum bandwidth."
```

Minor: I'd recommend either spelling out "respectively" or eliminating it.

```suggestion
      desc: "[in,out] Will contain the total number of bytes read from and written to memory as well as the current maximum bandwidth."
```